opentech/apply/funds/views.py

```diff
@@ -147,13 +158,24 @@ def get_context_data(self, **kwargs):
         )
 
 
-class SubmissionListView(AllActivityContextMixin, BaseAdminSubmissionsTable, DelegateableListView):
+@method_decorator(staff_required, name='dispatch')
```

Comment:
`ViewDispatcher` also handles the method decoration for you

Reply:
yes, decorator is not needed, I've corrected, thanks!